substra/sdk/client.py

```diff
@@ -81,7 +79,7 @@ def _update_permissions_field(data, permissions_field='permissions'):
 class Client(object):
 
     def __init__(self, config_path=None, profile_name=None, user_path=None,
-                 retry_timeout=DEFAULT_RETRY_TIMEOUT):
+                 token=None, retry_timeout=DEFAULT_RETRY_TIMEOUT):
```

Comment:
The input token seems unused here.

Reply:
Ah sorry, I'll remove it right away.

Reply:
Sorry*2, it is used, on lines 98-99

Reply:
It could be useful to accept a token as an input variable; maybe we could update the constructor method to handle it?

Reply:
👍 sorry ! I have been confused with the diff.